uniffi_bindgen/src/interface/mod.rs

```diff
@@ -614,6 +613,17 @@ impl Hash for ComponentInterface {
     }
 }
 
+fn get_or_insert_object<'a>(objects: &'a mut Vec<Object>, name: &str) -> &'a mut Object {
+    // The find-based way of writing this currently runs into a borrock error, so we use position
```

Comment:
```suggestion
    // The find-based way of writing this currently runs into a borrow error, so we use position
```

Reply:
It was supposed to be "borrowck", but I guess I should expand that to "borrow checker" 😄